pr_agent/algo/ai_handler.py

```diff
@@ -47,6 +46,13 @@ def __init__(self):
         except AttributeError as e:
             raise ValueError("OpenAI key is required") from e
 
+    @property
+    def deployment_id(self):
+        """
```

Comment:
**Suggestion:** Consider adding a docstring for the 'deployment_id' property to explain its purpose and usage.
```suggestion
    @property
    def deployment_id(self):
        """
        Property that gets the deployment ID from the settings.
        Returns None if the deployment ID is not set.
        """
        return get_settings().get("OPENAI.DEPLOYMENT_ID", None)
```